### PR TITLE
FluentMigrator helper library

### PR DESCRIPTION
**Pull request by smolyakoff**
Hi! I've created the helper library for [FluentMigrator](https://github.com/schambers/fluentmigrator) Would like to see it in the FAKE nuget package to simply run database migrations.

### CODE REVIEW

**Comment by dmorgan3405** [comment]
I am very interested in seeing this Module added to fake. Anything I can do to help with this? 

@forki / @smolyakoff anything I can do to help resolve the issues?

**Comment by forki** [comment]
it's marked WIP so I missed it. is it ready?

**Comment by smolyakoff** [comment]
@dmorgan3405 @forki Yes, this is still WIP, I wanted to create several public functions that wrap main *ExecuteDatabaseTask* function and expose the most frequently used functions of FluentMigrator. Hopefully, I will have some time next week to finish the module's interface. However, my build failed on the TravisCI and I didn't find a reason for that. 

@dmorgan3405 Feel free to contribute, probably you could find the reason for TravisCI failure or create more useful functions and document them

**Comment by dmorgan3405** [comment]
Sorry @forki, should of realized that it was a work in progress. Appreciate the quick response though. 

@smolyakoff I will take a look through what you have so far, and see if I can be of help.

**Comment by smolyakoff** [comment]
@forki Please, take a look if this pull request could be merged. Thanks!

**Comment by forki** [comment]
Cool will do tomorrow

File: src/app/Fake.FluentMigrator/FluentMigratorHelper.fs [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+    override this.Emphasize message = 
+        traceImportant (sprintf "[+] %s" message)
+
+    override this.Error message =
+        traceError (sprintf "!!! %s" message)
+
+    override this.Write (message, escaped) =
+        log message
+
+/// <summary>MS SQL Server driver version</summary>
```
Comment (forki) [inline src/app/Fake.FluentMigrator/FluentMigratorHelper.fs]: you don't need the summary tags here

File: src/app/Fake.FluentMigrator/FluentMigratorHelper.fs [hunk 2]
```diff
@@ -0,0 +1,308 @@
+﻿[<AutoOpen>]
```
Comment (forki) [inline src/app/Fake.FluentMigrator/FluentMigratorHelper.fs]: please do not use autoopen - we are trying to get rid of it everywhere.

**Comment by forki** [comment]
this looks pretty cool. the only thing left is a small tutorial which describes how to use it. this shouldn't be too hard since you already included the samples

**Comment by smolyakoff** [comment]
@forki, thanks for fast feedback! Where should I place the tutorial? Another .md file in *help* directory? I will make fixes according to your comments later today

**Comment by forki** [comment]
Yes. Just create a new md file. I will then link it from the website.

**Comment by smolyakoff** [comment]
@forki I've added a small tutorial to *help/fluentmigrator.md*. See if that works for you

**Comment by forki** [comment]
very cool. thx 

applause

**Pull request merged**